### PR TITLE
Issue #13213: Remove '//ok' comments from lambdabodylength

### PR DESCRIPTION
**Pull request by Acesif**
Issue #13213

Removed "//ok" comments from "InputLambdaBodyLengthMax.java" file 
and also removed respective suppressions

### CODE REVIEW

**Comment by romani** [comment]
Please fix commit message

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/InputLambdaBodyLengthMax.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public class InputLambdaBodyLengthMax {
                 .concat("3")
             )
         );
-        list.forEach(item -> // ok, 3 lines
+        list.forEach(item ->
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/InputLambdaBodyLengthMax.java:25]: Please revert this.

Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/InputLambdaBodyLengthMax.java:25]: Yes

**Comment by Acesif** [comment]
@MANISH-K-07 Can you please look into why there's a merge conflict, I can't seem to figure out why

**Comment by romani** [comment]
@Acesif , please do rebase on our latest master.
https://github.com/Acesif/checkstyle/commits/lambdabodylength/ your PR os based on march 23, but we have bunch of commits on march 25 - https://github.com/checkstyle/checkstyle/commits/master/.

here is now to rebase- https://checkstyle.org/beginning_development.html#Starting_Development

**Comment by Acesif** [comment]
> @Acesif , please do rebase on our latest master. https://github.com/Acesif/checkstyle/commits/lambdabodylength/ your PR os based on march 23, but we have bunch of commits on march 25 - https://github.com/checkstyle/checkstyle/commits/master/.
> 
> here is now to rebase- https://checkstyle.org/beginning_development.html#Starting_Development

Thanks, done.

File: config/checkstyle-input-suppressions.xml [hunk 2]
```diff
@@ -1370,7 +1370,21 @@
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]sizes[\\/]executablestatementcount[\\/]InputExecutableStatementCountMaxZero.java"/>
   <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]sizes[\\/]lambdabodylength[\\/]InputLambdaBodyLengthMax.java"/>
+            files="checks[\\/]sizes[\\/]filelength[\\/]InputFileLength2.java"/>
+  <suppress id="UnnecessaryOkComment"
```
Comment (romani) [inline config/checkstyle-input-suppressions.xml:1374]: No adding, only remal is allowed.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: config/checkstyle-input-suppressions.xml (hunk 2 above)
Comment (Acesif) [reply config/checkstyle-input-suppressions.xml:1374]: I didn't understand what you meant by that, can you please explain?

Comment (romani) [reply config/checkstyle-input-suppressions.xml:1374]: your PR add multiple suppressions.
It is not allowed.
We should remove suppressions.

**Review by romani — APPROVED** [review]
Ok to merge if ci pass

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**